src/app/pages/Collectibles/CollectiblePage.tsx

```diff
@@ -42,20 +58,51 @@ const CollectiblePage: FC<Props> = ({ assetSlug }) => {
 
   const creators = details?.creators ?? [];
 
+  const { isSelling, initiateSelling: onSellButtonClick } = useCollectibleSelling(assetSlug, details);
+
+  const onSendButtonClick = useCallback(() => navigate(`/send/${assetSlug}`), [assetSlug]);
+
+  const dispatch = useDispatch();
+  useInterval(() => void dispatch(loadCollectiblesDetailsActions.submit([assetSlug])), DETAILS_SYNC_INTERVAL, [
+    dispatch,
+    assetSlug
+  ]);
+
+  const offer = useMemo(() => {
+    const highestOffer = details?.highestOffer;
```

Comment:
do you handle the situation, when a highest offer is from the owner, but nft has other offers from other users?
I think you need to check this here